pkg/gwtestctl/client.go

```diff
@@ -129,17 +130,28 @@ func getClient(target string) (dataplane.ConfigServiceClient, func() error, erro
 	if err != nil {
 		return nil, nil, fmt.Errorf("parsing target: %w", err)
 	}
+	var conn *grpc.ClientConn
```

Comment:
you don't need to do any of those changes, just change a scheme from `unix-abstract` to `unix` if unix-abstract doesn't work for us